util/parser/ast_test.go

```diff
@@ -60,12 +52,14 @@ func (s *testASTSuite) TestSimpleCases(c *C) {
 		},
 	}
 
-	for _, t := range tests {
+	for _, testCase := range tests {
```

Comment:
```suggestion
	for _, test := range tests {
```

nit: if we range `tests`, we'd better name it `test`.